.changeset/light-chairs-happen.md

```diff
@@ -0,0 +1,27 @@
+---
+'@astrojs/react': minor
+'astro': minor
```

Comment:
This one can be a `patch`, since this is a change to an experimental flag. Probably, the same should be for `@astrojs/react`